src/arche/rules/coverage.py

```diff
@@ -98,3 +100,40 @@ def compare_scraped_fields(source_df: pd.DataFrame, target_df: pd.DataFrame) ->
         result.add_info(f"New - {', '.join(new_fields)}")
 
     return result
+
+
+def anomalies(jobs: List[Job]) -> Result:
```

Comment:
Not sure about the API.
In `Arche` we need to set only job keys, here we need job objects

Reply:
It's consistent with the internal API - you supply job for rules.
But I agree - I definitely want a wrapper to just supply keys.

Reply:
I thought about it the second time, and the convenience is more important. So now we need only keys, no wrappers.